import-map.json

```diff
@@ -1,19 +1,20 @@
 {
   "imports": {
-    "path": "./src/vendor/Path.ts",
```

Comment:
we'd prefer not re-ordering such things in an unrelated PR, but CI passed and whatever. :D

Reply:
No problem! I'll restore the previous order 😄